lib/awful/menu.lua

```diff
@@ -684,7 +684,7 @@ end
 -- -- a particular rule.
 -- -- Bound to a key, it can be used to select from dozens of terminals open on
 -- -- several tags.
--- -- When using @{rules.match_any} instead of @{rules.match},
+-- -- When using @{awful.rules.match_any} instead of @{awful.rules.match},
```

Comment:
This seems totally unrelated to screens. Did this chunk somehow end up in the wrong pull request? Should it just get its own PR and be merged to master basically immediately?

Reply:
Ok.

I am not sure why it didn't caused errors before. Isn't the point of `@` is to make those link explicit so they fail when the target doesn't exist? In that case, it didn't exist and it didn't fail until now.

Reply:
Well, the LDoc doc say:

> You can also put references in backticks, like `\`stdvars\``. This is commonly used in
> Markdown to indicate code, so it comes naturally when writing documents. The difference
> is that the backticked expression does not have to be a reference and then will appear
> in code style; with @ references you will get a warning for unrecognized symbols
> and the result will be rendered as '???'.

Out current docs say:

> -- When using ??? instead of ???,

https://awesomewm.org/apidoc/popups_and_bars/awful.menu.html

And when building things, I get the following warning from LDoc:

> usage:6: module not found: rules rules.match_any
>usage:6: module not found: rules rules.match

So everything seems to work as advertised for me. :-)

(But indeed: For the stable version the links work fine, so something changed somewhere; I'm to lazy to figure that one out right now)

Reply:
done